test/sql/alter.sql

```diff
@@ -89,3 +89,67 @@ SELECT * FROM _timescaledb_catalog.chunk WHERE id = 2;
 \set ON_ERROR_STOP 0
 ALTER TABLE public.new_chunk_name RENAME COLUMN time TO newtime;
 \set ON_ERROR_STOP 1
+
+\! mkdir /tmp/new_space
+CREATE TABLESPACE new_space LOCATION '/tmp/new_space';
```

Comment:
I did this because `psql` does not allow to put a relative path in `LOCATION` when creating a tablespace. Because installcheck is mainly used in travis testing I though this would be fine. But yes you are right, if the community wants to test, this could cause issues.
Maybe I can do some tricks with \! pwd to get the current dir but am not sure that will work.

Reply:
you can use `:TEST_TABLESPACE1_PATH` and `:TEST_TABLESPACE2_PATH` see the tablespace.sql test